docs/faq.md

```diff
@@ -129,3 +129,9 @@ pip install --upgrade "pip<10" "wheel<0.30.0" "setuptools<37"
 ```
 
 You can then [install watchmaker](installation.html) normally.
+
+## How do I get Watchmaker release/project notifications?
+
+Users can subscribe to the Watchmaker RSS feed which provides release and project notifications.  The Watchmaker RSS feed is <https://github.com/plus3it/watchmaker/releases.atom>
```

Comment:
This is really only "release" notifications, not project notifications (that's what the other option is for). How about:

> Users may use an RSS reader of their choice to subscribe to the Watchmaker Release feed to get notifications on Watchmaker releases. The Watchmaker RSS release feed is <https://github.com/plus3it/watchmaker/releases.atom>.